scripts/src/java/org/oppia/android/scripts/RegexPatternValidationCheck.kt

```diff
@@ -0,0 +1,161 @@
+package org.oppia.android.scripts
+
+import java.io.File
+import java.io.FileInputStream
+import org.oppia.android.app.model.FilenameChecks
+import org.oppia.android.app.model.FileContentChecks
+import kotlin.system.exitProcess
+
```

Comment:
no new line

Reply:
Shouldn't there be one between the import statements and the class declaration?

Reply:
Currently there are 2 empty lines 8th and 9th, so only 1 empty line is enough